schema.graphql

```diff
@@ -47,19 +47,40 @@ type Query {
   mostFollowedChannelsAllTime(limit: Int!): [ChannelFollowsInfo!]
 
   """Get most viewed list of categories"""
```

Comment:
Let's update the description in the code to `Get list of most viewed categories in a given time period`

Reply:
Let's do the analogous for other queries that I can't comment on :P